packages/headless/src/features/generated-answer/generated-answer-actions.test.ts

```diff
@@ -110,6 +111,26 @@ describe('generated answer', () => {
     });
   });
 
+  describe('#updateResponseFormat', () => {
+    const actionType = 'generatedAnswer/updateResponseFormat';
+
+    it('should accept a valid payload', () => {
+      const style = 'bullet';
```

Comment:
If you use the approach I suggested here https://github.com/coveo/ui-kit/pull/3279/files#diff-4aafd6f0a98338e8d58f437e99c2fee103893dc80ca0fb72632c9a0ce13d4dabR1, you could pick a random value from the `generatedAnswerStyle` array.

You could also test that an invalid value gets rejected.

Reply:
Hmm I don't love having randomness in the tests. I feel like they should run the same way every time and I'd hate to miss a bug because i won at test roulette haha. random coverage is arguably better than just covering 1 hard-coded option though 🤔. I might just loop

Reply:
Agreed, the full loop would be better given that there are very few values to iterate on anyways